### PR TITLE
Added 2 slides on nm and ldd in the compilation part

### LINKED ISSUE #145: Add a slide on nm, ldd and objdump
Both tools are mentioned in the exercise on the compilation chain, but we have no slide that mentions what these tools do and why/when they are needed.

### PR DESCRIPTION
**Pull request by sponce**
Closes #145

I did not address objdump, as this is more advanced and mostly superseeded by godbolt nowadays

### CODE REVIEW

File: talk/tools/compiling.tex [hunk 1]
```diff
@@ -84,6 +84,62 @@
   \end{block}
 \end{frame}
 
+\begin{frame}[fragile]
+  \frametitle{How to inspect object files ?}
```
Comment (bernhardmgruber) [inline talk/tools/compiling.tex:88]:
```suggestion
  \frametitle{How to inspect object files?}
```

File: talk/tools/compiling.tex [hunk 2]
```diff
@@ -84,6 +84,62 @@
...
+    \end{itemize}
+  \end{block}
+  \small
+  \begin{minted}{shell}
+> nm Struct.o
+                 U strlen
+                 U _Unwind_Resume
+000000000000008a T SlowToCopy::SlowToCopy(SlowToCopy const&)
+0000000000000000 T SlowToCopy::SlowToCopy()
+0000000000000064 T SlowToCopy::SlowToCopy(std::__cxx11::basic_string...
```
Comment (bernhardmgruber) [inline talk/tools/compiling.tex:114]:
```suggestion
> nm -C Struct.o
                 U strlen
                 U _Unwind_Resume
000000000000008a T SlowToCopy::SlowToCopy(SlowToCopy const&)
0000000000000000 T SlowToCopy::SlowToCopy()
0000000000000064 T SlowToCopy::SlowToCopy(std::__cxx11::basic_string...
```
The output looks demangled, so maybe you are missing the `-C` option here?

File: talk/tools/compiling.tex [hunk 3]
```diff
@@ -84,6 +84,62 @@
...
+  \end{minted}
+\end{frame}
+
+\begin{frame}[fragile]
+  \frametitle{How to inspect libraries/executables ?}
```
Comment (bernhardmgruber) [inline talk/tools/compiling.tex:119]:
```suggestion
  \frametitle{How to inspect libraries/executables?}
```

File: talk/tools/compiling.tex [hunk 4]
```diff
@@ -84,6 +84,62 @@
...
+    \item gives (recursive) list of libraries required by the given argument
+      \begin{itemize}
+      \item and if/where they are found in the current context
+      \end{itemize}
+    \item use \texttt{-r} to list missing symbols (mangled)
+    \end{itemize}
+  \end{block}
+  \small
+  \begin{minted}{shell}
+> ldd trypoly
```
Comment (bernhardmgruber) [inline talk/tools/compiling.tex:131]:
```suggestion
> ldd -r trypoly
```
I don't know whether you would need that here, but your text above says so.

**Comment by sponce** [comment]
All fixed. Thanks for the careful reading !

**Pull request merged**